site/content/blogs/release-v133.en.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'AWS Copilot v1.33: run local `--use-task-role`, and run local `depends_on` support
+twitter_title: 'AWS Copilot v1.33'
+image: ''
+image_alt: ''
+image_width: '1051'
+image_height: '747'
+---
+
+# 'AWS Copilot v1.33: run local `--use-task-role`, and run local `depends_on` support
+
+Posted On: January 8, 2024
+
+The AWS Copilot core team is announcing the Copilot v1.33 release.
+
+Our public [сommunity сhat](https://app.gitter.im/#/room/#aws_copilot-cli:gitter.im) has over 500 people participating, and our GitHub repository has over 3,100 stars on [GitHub](http://github.com/aws/copilot-cli/) 🚀.
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.33 brings big enhancements to help you develop more flexibly and efficiently:
+
+- **run local `--use-task-role`**: Elevate your local testing experience with the ECS Task Role using the new `--use-task-role` flag. [See detailed section](#use-ecs-task-role-for-copilot-run-local)
+- **run local `depends_on` support**:  Local run containers now respects `depends_on` in your service manifests. [See detailed section](#container-dependencies-support-for-copilot-run-local)
```

Comment:
Copilot v1.33 also upgrades all NLB security groups. This isn't exactly an enhancement but it needs a section somewhere.

Reply:
I put it in the release note! For a blogpost I think it's probably ok to only include features as for users they effectively don't need to do anything about it.